lib/bls.ex

```diff
@@ -18,6 +18,12 @@ defmodule Bls do
     :erlang.nif_error(:nif_not_loaded)
   end
 
+  @spec eth_fast_aggregate_verify([binary()], binary(), binary()) ::
+          {:ok, boolean} | {:error, any()}
```

Comment:
Rust's `String` is mapped into an Elixir binary
```suggestion
          {:ok, boolean} | {:error, binary()}
```

Reply:
ok :+1: , so it is also necessary to fix `fast_aggregate_verify` function